x-pack/plugins/maps/public/connected_components/layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -66,6 +73,32 @@ export function LayerSettings(props: Props) {
     }
   };
 
+  const renderIncludeInFitToBounds = () => {
+    if (!props.supportsFitToBounds) {
+      return null;
+    }
+    return (
+      <EuiFormRow display="columnCompressedSwitch">
+        <EuiToolTip
+          position="top"
+          content={i18n.translate('xpack.maps.layerPanel.settingsPanel.fittableFlagTooltip', {
+            defaultMessage: `Fit to data bounds adjusts your map extent to show all of your data. Layers that provide reference data may not need to be included in the data bounds. Use this option to exclude a layer from the fit to data bounds computation.`,
```

Comment:
Nit I still think "Fit to data bounds adjusts your map extent to show all of your data. Layers may provide reference data and should not be included in the fit to data bounds computation. Use this option to exclude a layer from fit to data bounds computation." reads better.